### ISSUE TITLE
[8.8 QA] Validate connector client Jira (Online & Server)

### ISSUE THREAD

**Issue by DianaJourdan** [elastic/connectors-python#860]
## Non-regression QA JIRA connector

**Note:** always test with the latest Build Candidate on Elastic Cloud, using the full Elastic stack

- [x] Start the whole stack from scratch and navigate to Enterprise Search -> Content
- [x] Check that no indices are shown in the top level Indices list
- [x] Click on "Create new index" - a new page is open where you can select an ingestion method
- [ ] Chose Build a connector option
- [x] Create an index with a valid name and Universal language
-------

- [x] Connector name and description are editable on the Configurations page
- [x] Connector can be deleted from the Indices page
- [x] Connector can be deleted from the Indices page and it can be recreated with the same name after
- [x] Pull connectors repository, run `make install` but do not run connector yet
- [x] Create an index with a valid name and non-universal language and check language was saved correctly
- [x] Verify that you are redirected to "configuration" page where you can create an api key and can copy connector id / whole section of config into the connector
- [x] Update connector configuration with the api_key and connector_id, plus choose a service_type to test and set it in config
- [x] Start the connector by running `make run` - verify that it starts and does not actually do anything yet
- [x] Wait for the Kibana page with the connector configuration to update and verify that it's possible to edit connector configuration now
- [x] Edit and save connector configuration, then reload the page and verify that configuration is properly saved
- [x] Click on "Set schedule and sync" and verify that you're redirected to the scheduling tab
- [x] Enable scheduling for frequency = every hour and save schedule; refresh the page and verify that the changes were stored
- [x] Switch to the connector and wait for a minute or two, verify that connector starts to ingest data
- [x] Verify that the data from the connector appears in the expected index
- [ ] Verify that on the connector overview page index information is updated properly, showing expected number of documents and new index size
- [x] Verify that on the connector overview page "Document Count" is updated to reflect the number of documents in the index
- [x] Verify that you can see ingested documents in `documents` tab
- [x] Verify that index mappings are correct on the `index mappings` tab

Record a short demo showing the connectors' configuration and that there were documents ingested

**Comment by khusbu-crest** [comment]
@DianaJourdan
I have validated the above checklist for connector client Jira and overall it seems working fine.
Below are a few points, in the checklist, which is a bit different than what is mentioned:

For --> Chose Build a connector option --> We have checked it via the Use a Connector > Customised Connector option (as discussed in Slack with @danajuratoni )

For -->Verify that on the connector overview page index information is updated properly, showing the expected number of documents and new index size --> The Index size is not displayed on the Elasticsearch Indices overview page. We are able to check the same from Index Management Page as it is displayed there.

NOTE: I am unable to mark the checklist items as done, therefore adding the comment for the validations done for the connector.
